locales/en/app.ftl

```diff
@@ -430,8 +430,8 @@ what-data= What data was compromised:
 overview= Overview:
 
 
-sensitive-sites= How does Firefox Monitor treat sensitive sites?
-sensitive-sites-copy= Firefox Monitor reveals accounts associated with these 
+sensitive-sites= How does {-product-name} treat sensitive sites?
+sensitive-sites-copy= {-product-name} reveals accounts associated with these 
```

Comment:
I know it's early for me to review this, but there should be a space before and after the equal sign (right now the file is inconsistent) ;-)